molgenis-data/src/main/java/org/molgenis/data/transaction/TransactionConfig.java

```diff
@@ -16,10 +17,14 @@
 	@Autowired
 	private MolgenisTransactionManager transactionManager;
 
+	// This cannot be a app setting because apps settings are not loaded yet when we use it
+	@Value("${use.transaction.log:true}")
```

Comment:
Transaction log logs wich transactions per user and is used for checking concurrent modifications, is not always needed and takes a lot of time